packages/common/src/utils/formatters.ts

```diff
@@ -9,6 +9,11 @@ export const formatNaiveDate = (date?: Date | null): string | null => {
   else return null;
 };
 
+export const formatNaiveDateTime = (date?: Date | null): string | null => {
```

Comment:
ah.. NaiveDateTime has time but not timezone.. I see now. Although I thought that timezone was important 🤔 
not a real comment, just saying stuff out loud. I understand that the NaiveDateTime is going away.. so nothing to do here